src/DogImage.js

```diff
@@ -0,0 +1,12 @@
+function DogImage( {dogUrl, breed} ) {
+    return( 
+        <div className="dogImage">
+            <h2>{breed}</h2>
+            <div className="imgContainer">
+                <img src={dogUrl} alt={`A really cute doggie! ${breed}`} />
+            </div>
```

Comment:
In chrome I can see an img error on page load near "a really cute doggie" - perhaps having a ternary operator to have a placeholder picture and then have the ability to display the dogs when the user selects an option would be helpful. But in firefox it looks good.